pyevmasm/evmasm.py

```diff
@@ -24,7 +27,7 @@
         '0x606040526002610100'
         >>> disassemble_hex('0x606040526002610100')
         'PUSH1 0x60\\nBLOCKHASH\\nMSTORE\\nPUSH1 0x2\\nPUSH2 0x100'
-        
+
```

Comment:
Hmm... I wonder if doctest would catch this string. If no, maybe it catches module docstrings and so moving it to a module docstring could let us launch doctest over it if we don't do it yet.

Idk, loud thinking, not relevant to the PR.

@computerality (marking u so u can decide if we want it and make an issue or skip it)

Reply:
made a separate issue to track this idea